recipes-support/snapd/snapd_2.47.1.bb

```diff
@@ -43,7 +43,8 @@ RDEPENDS_${PN} += "		\
 S = "${WORKDIR}/${PN}-${PV}"
 
 EXTRA_OECONF += "			\
-	--disable-apparmor		\
+	--enable-apparmor		\
```

Comment:
AppArmor requires non default kernel options, and assumes you actually have apparmor runtime which is provided by meta-security. This is why it was kept optional, and can always be overridden when you bbappends. I think the `--disable-apparmor` needs to be restored here.